internal/run/source.go

```diff
@@ -711,7 +719,7 @@ func overlayDocument(ctx context.Context, schema string, overlayFiles []string,
 		}
 
 		// YamlOut param needs to be based on the eventual output file
-		if err := overlay.Apply(currentBase, overlayFile, utils.HasYAMLExt(outFile), tempOutFile, false, false); err != nil {
+		if err := overlay.Apply(currentBase, overlayFile, utils.HasYAMLExt(outFile), tempOutFile, false, false); err != nil && strings.Contains(err.Error(), "overlay must define at least one action") {
```

Comment:
This looks to swallow all errors except for the one detecting the No-op overlay, which doesn't seem right?

We could remove the no-op error.